packages/trader/src/Modules/MT5/Containers/mt5-password-manager-modal.jsx

```diff
@@ -15,19 +15,49 @@ import {
 import { Field, Form, Formik } from 'formik';
 import PropTypes from 'prop-types';
 import React from 'react';
-import { getDerivComLink } from '@deriv/shared/utils/url';
 import { localize, Localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 import MT5Store from 'Stores/Modules/MT5/mt5-store';
 import { validLength, validPassword } from 'Utils/Validator/declarative-validation-rules';
 
+const CountdownComponent = ({ count_from = 60, onTimeout }) => {
+    const [count, setCount] = React.useState(count_from);
+    if (count === 0) {
+        onTimeout();
```

Comment:
should this be in the `useEffect` with [count]? if we only want `onTimeout` to fire once when the count reaches 0 🤔

Reply:
That is a great idea. Will try it out.